### ISSUE TITLE
Submitting Prework: Tips Calculator Application

### ISSUE THREAD

**Issue by dongtanhuy** [dongtanhuy/Prework-Tips-Application#1]
Here is my completed prework! Thank you
/cc @coderschoolreview

**Comment by coderschoolreview** [comment]
### Thank you for your Swift submission, dongtanhuy!

 :bow: Everything looks good with the format of your submission. We'll be reviewing your submission soon!

**Comment by coderschoolreview** [comment]
:+1:  Great job with the Tip Calculator! 

### Quick Comments:
- I like the app background with the option to change it :smile:  
- It seems like the default percentage in Setting was not implemented. **This is required. Pls update your code asap.**
- Good usage of `NSUserDefaults`. This is one of the methods to pass the data from multiple viewController. We will learn few more ways to do that. 
- If you were able to save the tipPercentage, why didn't you save the billAmount? (one of the optional). It should be exactly the same.
- It's OK to use `String(format: "$%.2f", tip)` in this case, but if you change the language to Vietnamese for example, we don't use 2 digits after decimal point and the $ sign, right? I recommend to try `NSNumberFormatter` and `CurrencyStyle`
- You shouldn't let the user enter invalid number, should you? I can enter bunch of leading zeros, and the dots. I can also paste some text in the text field. 

<img width="322" alt="screen shot 2016-02-22 at 1 11 59 pm" src="https://cloud.githubusercontent.com/assets/15702670/13210945/bd81be1c-d966-11e5-8bab-fd21ed603226.png">
- Your app is not able to remember the bill amount across app restarts. Whenever I restart, it shows 0 again.
- I don't see the keyboard show up when the add first launch. You can show the keyboard immediately when app starts by using `becomeFirstResponder`

### Optional Suggestions to Improve
- Look into the `UIView.animateWithDuration` functions to add animations.

Bottom line, you need to complete the requirement for the **default percentage in setting page**. Others are optional.

**Comment by coderschoolreview** [comment]
Hi @dongtanhuy , any update on the **default percentage in setting page.**?

**Comment by dongtanhuy** [comment]
Sorry, I am busy now. I will update my code as soon as posible before 28th Feb

**Comment by dongtanhuy** [comment]
Hi, @coderschoolreview, I have just submitted my code to update on **the default percentage in setting page**!
Thanks!
